deploy/dependencies/kustomization.yaml

```diff
@@ -34,8 +37,20 @@ resources:
 - https://raw.githubusercontent.com/rhobs/obo-prometheus-operator/v0.60.0-rhobs1/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml
 - https://raw.githubusercontent.com/rhobs/obo-prometheus-operator/v0.60.0-rhobs1/example/rbac/prometheus-operator/prometheus-operator-service-account.yaml
 
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+# Admission Webhook Deployment
+- https://raw.githubusercontent.com/rhobs/obo-prometheus-operator/v0.60.0-rhobs1/example/admission-webhook/deployment.yaml
+- https://raw.githubusercontent.com/rhobs/obo-prometheus-operator/v0.60.0-rhobs1/example/admission-webhook/service-account.yaml
+# NOTE: a service although automatically created by OLM for webhooks still
+# requires admission-webhook/service as the port generated by OLM uses 443 as
+# opposed to 8443 used by the admission-webhook
```

Comment:
Thanks for the heads up, @simonpasquier . We will have to `kustomize` the deployment a bit more I guess since OLM injects the tls certs. 
The service port 443 will be good change but isn't there a restriction on running on ports < 1024 ? Do we need additional permissions ?

Reply:
> The service port 443 will be good change but isn't there a restriction on running on ports < 1024 ? Do we need additional permissions ?

not that I know.

Reply:
> lgtm. Let's make sure to document that the admission webhook's TLS materials + webhook configurations (caBundle) are fully controlled by OLM [1].
> 
> [1] https://sdk.operatorframework.io/docs/building-operators/golang/webhook/#generate-webhook-manifests-and-enable-webhook-deployment

@simonpasquier Any thoughts on where we want to start documenting ? `docs/design/olm.md` ?

Reply:
Sounds like a good place to start 👍